src/net/sourceforge/kolmafia/persistence/ConsumablesDatabase.java

```diff
@@ -329,73 +265,50 @@ private static void saveConsumptionValues(String[] data, Map<String, Integer> ma
     String holiday = HolidayDatabase.getHoliday();
     boolean isBorisDay = (holiday.contains("Feast of Boris") || holiday.contains("Drunksgiving"));
 
-    ConsumablesDatabase.levelReqByName.put(name, level);
-
     // Some items different on Feast of Boris
     if (isBorisDay) {
       switch (name) {
         case "cranberries" -> {
-          ConsumablesDatabase.qualityByName.put(name, ConsumableQuality.GOOD);
-          ConsumablesDatabase.saveAdventureRange(name, unitCost, "2-4");
+          quality = ConsumableQuality.GOOD;
+          adventures = "2-4";
         }
         case "redrum" -> {
-          ConsumablesDatabase.qualityByName.put(name, ConsumableQuality.GOOD);
-          ConsumablesDatabase.saveAdventureRange(name, unitCost, "5-9");
+          quality = ConsumableQuality.GOOD;
+          adventures = "5-9";
         }
         case "vodka and cranberry" -> {
-          ConsumablesDatabase.qualityByName.put(name, ConsumableQuality.GOOD);
-          ConsumablesDatabase.saveAdventureRange(name, unitCost, "6-9");
-        }
-        default -> {
-          ConsumablesDatabase.qualityByName.put(name, ConsumableQuality.find(quality));
-          ConsumablesDatabase.saveAdventureRange(name, unitCost, adventures);
+          quality = ConsumableQuality.GOOD;
```

Comment:
This doesn't look right

Reply:
why not? The lines are the same and in the same sequence, I just removed the default case and have it fall through below to reduce code duplication.

Reply:
Am I reading this wrong or is the default for items to be good here? I was reviewing from mobile

Reply:
This is under "vodka and cranberry".

But it would be good to have a test that non-Boris items have the same quality / adventures as normal on Boris.

Reply:
Added a couple lines to the test for this.